src/utils/_opaciy.scss

```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
```

Comment:
Typo in the filename: `src/utils/_opaciy.scss`

Reply:
thanks, fixed!